### PR TITLE
Portfolio Setup and Mobile First: Add index.html, style.css and Linters

### PR DESCRIPTION
**Pull request by rmiyoyo**
# Work Done 
- Added the ```index.html```, ```style.css``` files and images for the portfolio site
- Added the HTML/CSS linters to check for errors.
- Implemented flexbox

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: Setup and Mobile First</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;400;600;700&display=swap">
+</head>
+<body>
+  <div class="container">
+    <nav>
+      <a href="#" class="logo">My Logo</a>
```
Comment (OyePriscilla) [inline index.html:13]: - [ ] Please kindly bring the logo out of the `nav` tag, a logo is not supposed to be part of the navigation links. 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: Setup and Mobile First</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;400;600;700&display=swap">
+</head>
+<body>
+  <div class="container">
```
Comment (OyePriscilla) [inline index.html:11]: - [ ] Please for semantics purpose, kindly use the `header` tag in place of the div tag. The [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) header element represents introductory content, typically a group of introductory or navigational aids. 👍

File: index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      <p class="social-media">LET'S CONNECT</p>
+      <div class="social-icons">
+        <a href="#"><img src="./images/icons/twitter.svg" alt="Twitter"></a>
+        <a href="#"><img src="./images/icons/Linkedin.png" alt="LinkedIn"></a>
+        <a href="#"><img src="./images/icons/m.svg" alt="Medium"></a>
+        <a href="#"><img src="./images/icons/github.svg" alt="GitHub"></a>
+        <a href="#"><img src="./images/icons/random.svg" alt="Random"></a>
+      </div>
+    </div>
+  </div>
```
Comment (OyePriscilla) [inline index.html]: - [ ]  The container of this section suppose to be a `section` not `div` tag. It represents a generic standalone section of the document. 👍

File: index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    </div>
+  
+    <div class="connect text-left">
+      <p class="social-media">LET'S CONNECT</p>
+      <div class="social-icons">
+        <a href="#"><img src="./images/icons/twitter.svg" alt="Twitter"></a>
+        <a href="#"><img src="./images/icons/Linkedin.png" alt="LinkedIn"></a>
+        <a href="#"><img src="./images/icons/m.svg" alt="Medium"></a>
+        <a href="#"><img src="./images/icons/github.svg" alt="GitHub"></a>
+        <a href="#"><img src="./images/icons/random.svg" alt="Random"></a>
```
Comment (OyePriscilla) [inline index.html]: - [ ]  Kindly use a `ul` tag to group the list of these images, and `li` per icon; each icon has a link (a tag). Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using `href`. 👍

File: style.css [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+body {
+  margin: 0;
+  background-image: url("./images/bg.png");
+  background-repeat: no-repeat;
+  background-position: center 76px;
+  background-size: cover;
+  font-family: 'Poppins', sans-serif;
+}
```
Comment (OyePriscilla) [inline style.css]: - [ ] Please kindly take note of the general structure of your design when compared to the given template. I couldn't see the `left-border-bottom` radius and more so, all the background images are not showing. Please ensure to align all the elements on the page according to the given template. Our ability to reproduce perfectly a given design shows that we pay attention to details. 👍 

## Figma design
Untitled

## Your design
Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @rmiyoyo 👍 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Linters check passed ✔️
Observed GitHub Flow ✔️
Content is aligned with flexbox ✔️
Used images and background-image to enhance the look of the website ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rmiyoyo** [comment]
# Work Done After Review
- Removed logo from ```nav``` tag
- Adjusted the alignment of sections to be uniform
- Added ```header``` tag after the main container ```div```
- Replaced ```div``` with ```section``` tag in the social media icons
- Used an unordered list in the social media section
- Revised general structure of the design to include all background images and alignments

**Review by Shedrack-Sunday — APPROVED** [review]
# STATUS: APPROVED ♻️♻️
Hi @rmiyoyo 

To highlight 👍🏽
✔️ Proper use Semantics
✔️ No linter errors

Your project is complete! There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**